.ci/stack-validate-pipeline-variables.yml

```diff
@@ -0,0 +1,91 @@
+#
+# Repos
+#
+
+#. git_repository (required): git@github.com:my/repo_to_validate.git
+#+ Git repository URL containing the stack to validate.
+git_repository: git@github.com:cycloid-community-catalog/stack-ses.git
+
+#. git_branch (required): master
+#+ Branch of the stack to validate Git repository.
+git_branch: master
+
+# git status
+git_repository_name: cycloid-community-catalog/stack-ses
+git_access_token: ((github_access_token.data))
+
+#. git_key (required): ((git.ssh_key))
+#+ SSH key pair to fetch stack source code Git repository.
+git_key: ((git_github.ssh_key))
+
+#. molecule_env_vars (required): dict
```

Comment:
Just a note to think to remove this when you will do the dedicated pipeline for terraform only (same for *ansible* under)